Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM golang:alpine as builder
```

Comment:
Let's keep this docker file in `docker/node` folder, since this is specific to node applications